dex/src/main/scala/com/wavesplatform/dex/tool/AssetCacheCleaner.scala

```diff
@@ -0,0 +1,27 @@
+package com.wavesplatform.dex.tool
+
+import cats.Id
+import com.wavesplatform.dex.cli.ErrorOr
+import com.wavesplatform.dex.db.DbKeys
+import com.wavesplatform.dex.db.leveldb.{openDb, LevelDb}
+import cats.syntax.either._
+import com.wavesplatform.dex.error.Implicits.ThrowableOps
+import org.iq80.leveldb.DB
+
+object AssetCacheCleaner {
+
+  def cleanAssets(dataDirectory: String): ErrorOr[Unit] = Either.catchNonFatal {
+    val db: DB = openDb(dataDirectory)
+    LevelDb.sync(db)
+  }.flatMap(cleanAssets)
+    .leftMap(ex => s"Error during removing assets: ${ex.getWithStackTrace}")
```

Comment:
Here is a problem: we don't close `db`. Also, I this it'd better to move this function to `WavesDexCli`, because it is a very specific function, that will be used on `WavesDexCli` only. Also it is okay to throw an exception in `WavesDexCli`.

Suggest to:
1. Move this code `WavesDexCli`, e.g. `def withLevelDb(dataDirectory: String)(f: LevelDb[Id] => Unit): Unit`
2. Wrap `db` in `Using`